dimscord/restapi/message.nim

```diff
@@ -176,15 +177,17 @@ proc editMessage*(api: RestApi, channel_id, message_id: string;
         endpointChannelMessages(channel_id, message_id),
         $payload,
         mp = mpd
-    )).newMessage(api)
+    )).newMessage
+    result.setContext(api)

```

Comment:
but we already know that api is not nil, the use of the setContext is useless.
should just do `result.ctx = api`